client/js/lounge.js

```diff
@@ -742,13 +757,29 @@ $(function() {
 		}
 	}
 
-	sidebar.on("click", ".chan, button", function() {
+	sidebar.on("click", ".chan, button", function(e, data) {
 		var self = $(this);
 		var target = self.data("target");
 		if (!target) {
 			return;
 		}
 
+		if (!data || data.pushState !== false) {
```

Comment:
Yep, but, knowing myself, if I write a boolean expression like this it's usually for a reason. I don't remember what that reason was though.